### ISSUE TITLE
Cannot play music or navigate app for 30 seconds after opening. 

### ISSUE THREAD

**Issue by Mattevt** [timusus/Shuttle#357]
**Shuttle version:**
 
 v2.0.4
 
**Device, OS:**
 
Motorola 5gs

**Description of bug:**
 
Extensive delay from time of app appearance to when it becomes usable. Image link below. 
 
**Steps to reproduce:**
 
 1. Close app completely 
 2. Open app
 
**Expected outcome:**
 
Application opens reasonably quickly with almost immediate ability to navigate interface/play tracks. 
 
**Observations/Actual Result:**

Application opens to blank page. Unable to take any action for ~30 seconds.

https://imgur.com/a/KJkQUmP

**Comment by timusus** [comment]
Thanks @Mattevt.

Can you please list your OS version as well?

Are you familiar enough with Android that you'd be able to capture some logs? It would be interesting to see what is going on that causes the ~30 second delay. This is definitely unusual, though I don't think you're the only one experiencing this issue.

**Comment by Mattevt** [comment]
Updated with OS. I could take logcats before, but I don't run a Windows machine anymore and I haven't tried adb on Ubuntu.

**Comment by timusus** [comment]
Relates to / possible duplicate of #79

**Comment by Mattevt** [comment]
Yes, that's it. I'm using an SD card with a large library as well, though I didn't notice he issue until relatively recently.

I have had the app crash when playing a playlist, but not regularly like OP in the reddit post. It could easily be an OS problem.

**Comment by 4ut0-M-4t** [comment]
**OS/version**: Android 6.0.1
**Shuttle version**: 2.0.6b3
---
I hope this will be of any assistance to you:
[Shuttle_start_delay_2018-09-12.txt](https://github.com/timusus/Shuttle/files/2377365/Shuttle_start_delay_2018-09-12.txt)
Unfortunately it's not a full log, but it might still help you get an idea.
By the way, I am experiencing near a minute delay.

**Comment by timusus** [comment]
Thanks @4ut0-M-4t . Unfortunately, there's not much there at all. The warnings there are just saying that Shuttle can't fix the missing year tag in the media store for some mp4 tracks. I'd need to see more of the logs, not filtered by the shuttle package, from just before the app is launched until the files are loaded, to see if there are any system calls explaining what's causing the delay.
